server/api/core/groups.js

```diff
@@ -76,6 +76,7 @@ function getDefaultGroupRoles() {
 
   // we're making a Shop Manager default group that have all roles except the owner role
   const shopManagerRoles = ownerRoles.filter((role) => role !== "owner" && role !== "admin");
+  shopManagerRoles.push("shopSettings");
```

Comment:
this line feels odd. I can't quite put my finger on why, but I guess I don't understand why `"shopSettings"` is such a unique role that it is called out explicitly here where no other roles are.

Reply:
I sort of agree with you, but we don't really have a place (or alternately I did not find the place) where we are setting these defaults. The existing set up is just "everything that owner gets minus `owner` and `admin`" which also seems a little, weird?

Plus this is basically the only place where a shop owner is defined, so it would seem to make sense to assign any roles that are explicitly "shop owner only" here?